modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -1119,6 +1183,8 @@ class cv::gimpl::GStreamingExecutor::Synchronizer final {
     // Start a thread which will handle the synchronization.
     // Do nothing if synchronization is not needed
     void start() {
+        GAPI_LOG_INFO(nullptr, "try to start syncActorThread for emitters count: " <<
+                                m_synchronized_emitters.size());
         if (m_synchronized_emitters.size() != 0) {
```

Comment:
Troubleshooting & Logging BKMs:

print out COUNT of object BEFORE condition serve TWO purposes:
-to check actual objects count :)
-to find out is the next condition true or false

So we use only one logger row and do not require to logging approaches for different if-branches